apps/mull-ui/src/app/components/event-page-info/event-page-info.scss

```diff
@@ -0,0 +1,28 @@
+@import 'variables';
+
+.event-page-info-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.info-row {
+  display: flex;
+  padding: 0.5rem 1rem;
```

Comment:
I find the rows could use more breathing room. The spacing in the mockups is not consistent so it's hard to say exactly what we should use. I suggest 1.25rem.
```suggestion
  padding: 1.25rem 1rem;
```

Reply:
Nice catch, I do agree that some more padding is needed, however I feel that 1.25 is excessive from my testing. I bumped up the padding to 1rem:

implementation:
![image](https://user-images.githubusercontent.com/23544999/99762323-3d950680-2ac6-11eb-8770-3b6537ae477b.png)

mockup:
![image](https://user-images.githubusercontent.com/23544999/99762405-6f0dd200-2ac6-11eb-9186-535a316da34b.png)